apps/generic/ui/aboutdialog.cpp

```diff
@@ -61,7 +61,7 @@ AboutDialog::AboutDialog(QWidget *parent) :
     vlayout->addWidget(label1);
 
     QLabel *label2 = new QLabel(tr("Thanks to") + "\n" \
-                                "XBMC Development Team\n");
+                                "Kodi Development Team\n");
```

Comment:
Hi Robert, thank you for your comments. :)
According to http://kodi.wiki/view/Team_Kodi it is now "Team Kodi":
> Team Kodi is the formal group of people who develop and maintain XBMC/Kodi, the Kodi websites, and/or other supporting roles. 

But to be honest, I did not check this before. ;)